packages/@n8n/nodes-langchain/nodes/memory/MemoryMongoDBChat/MemoryMongoDBChat.node.ts

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable n8n-nodes-base/node-dirname-against-convention */
+import { MongoDBChatMessageHistory } from '@langchain/mongodb';
+import { BufferMemory, BufferWindowMemory } from 'langchain/memory';
+import { MongoClient } from 'mongodb';
+import type {
+	ISupplyDataFunctions,
+	INodeType,
+	INodeTypeDescription,
+	SupplyData,
+} from 'n8n-workflow';
+import { NodeConnectionType, NodeOperationError } from 'n8n-workflow';
+
+import { getSessionId } from '@utils/helpers';
+import { logWrapper } from '@utils/logWrapper';
+import { getConnectionHintNoticeField } from '@utils/sharedFields';
+
+import {
+	sessionIdOption,
+	sessionKeyProperty,
+	contextWindowLengthProperty,
+	expressionSessionKeyProperty,
+} from '../descriptions';
+
+export class MemoryMongoDBChat implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'MongoDB Chat Memory',
+		name: 'memoryMongoDBChat',
+		icon: 'file:mongodb.svg',
+		group: ['transform'],
+		version: [1, 1.1, 1.2, 1.3],
```

Comment:
since this is new node, we don't need to have all these versions, and can just put everything under version `1` for now.